test/runtests.jl

```diff
@@ -294,3 +294,6 @@ end
 include("struct.jl")
 include("json.jl")
 include("numbers.jl")
+@static if VERSION ≥ v"1.8"
+    @testset "Optimization test with JET" include("optimization.jl")
+end
```

Comment:
If you don't like adding this dependency on JET, let me know. Sometimes, there are discrepancies between JET and Julia's nightly version, causing occasional failures in the nightly CI. Generally there is no problem because I regularly update JET.

Reply:
I don't mind the dependency; I've generally had a positive experience my using it as a test dependency